R/grouped_epi_archive.R

```diff
@@ -366,6 +370,9 @@ grouped_epi_archive =
                                       .keep = TRUE)
               )
             })
+            x <- rbindlist(x) %>% setDF() %>% as_tibble() %>%
+              # Reconstruct groups
+              group_by(!!!groups(x[[1L]]), .drop = attr(attr(x[[1L]], "groups"), ".drop"))
```

Comment:
Is there a better way to carry forward groups and `.drop` setting? This seems awkward.

Reply:
Oh, I guess I can access private$drop and private$vars here